### PR TITLE
docs: fixes options section anchor link

### PR DESCRIPTION
**Pull request by abea**
- [✔️] Tests and linter pass
- [✔️] Code coverage does not decrease (if any source code was changed)
- [✔️] Appropriate docs were updated (if necessary)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -290,7 +290,7 @@ const drive = google.drive({
 });
 ```
 
-See the [Options section][options] for more information.
+See the [Options section][request-options] for more information.
```
Comment (bcoe) [inline README.md]: I think this should be `[Options section](#request-options)`?

Comment (abea) [reply README.md]: I thought so too, but since the original worked like that I assumed it was a valid style. I'll make that update.

**Comment by JustinBeckwith** [comment]
Thanks for the fix!  I looked into this though... and it looks like the link is working today.  I'm going to go ahead and close this out for now.

**Comment by abea** [comment]
It's still not working for me in Firefox or Chrome, taking me to https://github.com/googleapis/google-api-nodejs-client/tree/master#options, which just sits at the top of the page since that's no longer a heading. I'd encourage you to check again, but up to you.

**Comment by JustinBeckwith** [comment]
And I stand corrected ... you were totally right 🙃 Thanks for the fix!

**Pull request merged**